news/fips-cve.md

```diff
@@ -0,0 +1,51 @@
+---
+breadcrumb: CVEs and FIPS
+---
+# CVEs and the FIPS provider
+
+After the release of OpenSSL 3.0.0, several CVEs have been identified
+and resolved.  While the majority of these vulnerabilities are unrelated
+to the validated FIPS provider, a few of them are applicable.  This table
+lists all of the CVEs issued since the FIPS provider's release and their
+relevance to it:
+
+<!--
+  -- Pandoc allocates horizontal space equally, we don't want that so override
+  -- This, however, removes spacing between table columns so we force some
+  -- using non-breaking spaces.
+  -->
```

Comment:
Actually, it bases the size proportions on the table header underlining (i.e. line 24 below).

When I tried expanding the first underlining (after removing the style), i.e. to this:

```
------------- | :-:   | :-: | :--
```

I got a different spacing:

```
<colgroup>
<col style="width: 59%" />
<col style="width: 13%" />
<col style="width: 13%" />
<col style="width: 13%" />
</colgroup>
```

Reply:
Thanks, I wasn't aware of this and didn't find any documentation explaining it.

Reply:
I just looked through the user guide, and indeed, there isn't much about it in there (and what is there is incredibly vague)